### PR TITLE
doc: Add info how to get libSkiaSharp.dylib

### PR DESCRIPTION
**Pull request by TopperDEL**
## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the current behavior?

Building the native dylib for Mac needs an extra step - the build instruction mentions this but on a different page. 

## What is the new behavior?

This change highlights a bit what needs to be done before building succeeds.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.MacOS/README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Skia Host for macOS
 
+## Preparation
+
+Before you can build you need the libSkiaSharp.dylib.
+```bash
+cd UnoNativeMac
+./getSkiaSharpDylib.sh
+```
+See /UnoNativeMac/README.md for more info.
```
Comment (spouliot) [inline src/Uno.UI.Runtime.Skia.MacOS/README.md:10]: This could be a link to the other README.md

**Review by spouliot — APPROVED** [review]
Thanks!

It's done automatically when you build `SampleApp.Skia.Generic` (or the whole Uno) so I forgot to document it for a standalone compilation.

File: src/Uno.UI.Runtime.Skia.MacOS/README.md (hunk 1 above)
Comment (TopperDEL) [reply src/Uno.UI.Runtime.Skia.MacOS/README.md]: You're right - done

**Comment by TopperDEL** [comment]
I've added the link, the PR might be merged now

**Review by spouliot — APPROVED** [review]

**Comment by morning4coffe-dev** [comment]
@TopperDEL Thank you for your contribution! Can you please Reword your commits with https://www.conventionalcommits.org/en/v1.0.0/ both are failing on this 
image

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**